proxy/envoy/testdata/cb-policy.yaml.golden

```diff
@@ -2,10 +2,10 @@ destination: world.default.svc.cluster.local
 circuit_breaker:
   simple_cb:
     max_connections: 100
-    sleep_window_seconds: 15.5
+    sleep_window: 15.5s
```

Comment:
@rkpagadala these are used for testing the envoy config generation.  I figured it would be useful to test one case using `15.5s` and another `15500ms` to verify the desired config is achieved with both formats.